README.md

```diff
@@ -379,6 +379,72 @@ parameters.completionBlockQueue = dispatch_get_main_queue();
 }];
 ```
 
+### Multiple Account Mode
+
+MSAL also provides a public API to query multiple accounts, granted that they exist in the MSAL cache. 
+1) First import MSALSilentTokenParameters.h in addition to the other core public libraries that are required to use the public APIs.
```

Comment:
I think developers should just import the umbrella header (MSAL/MSAL.h) and it should be part of the setup steps.